lib/browser_launcher.dart

```diff
@@ -0,0 +1,5 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Use dart project standard headers.

Copy from here https://github.com/dart-lang/webdev/blob/master/dwds/lib/service.dart

Reply:
done. Also added an AUTHORS file since the standard header references it.